### PR TITLE
Attempts to band aid spine wagging

### PR DESCRIPTION
**Pull request by Majkl-J**
## About The Pull Request

closes: #252

### Quick rundown of what is broken
For some reason, someone somewhere deleted the Insert proc call for lizard tails. I have looked through about 50 different files, ran through the last 3 months of PRs that touched organ/customization code, and found nothing out of place, be it for my inexperience or influence of a higher power.

Thus I have put together this QoL quickfix: Spines are now checked again when trying to wag, which works.

(Please someone who knows more about this try to point me in the right direction so I don't have to do this)

- [x] Searched for the error
- [x] Found the issue
- [x] Cried a lot trying to fix it
- [x] Tested the fix
- [ ] Make the spines not disappear

## Changelog
:cl:
fix: Lizard spines no longer disconnect from the rest of the body upon wagging
/:cl:

### CODE REVIEW

File: code/modules/surgery/organs/external/tails.dm [hunk 1]
```diff
@@ -117,11 +117,14 @@
...
 
-/obj/item/organ/external/tail/lizard/Insert(mob/living/carbon/reciever, special, drop_if_replaced)
+//BUBBER FIX START
+//This edit atempts to bandaid the spine wagging issue and should be overriden the moment someone else figures it out
+/obj/item/organ/external/tail/lizard/Insert(mob/living/carbon/receiver, special, drop_if_replaced)
 	. = ..()
-	if(.)
-		paired_spines = ownerlimb.owner.get_organ_slot(ORGAN_SLOT_EXTERNAL_SPINES)
-		paired_spines?.paired_tail = src
+	paired_spines = ownerlimb.owner.get_organ_slot(ORGAN_SLOT_EXTERNAL_SPINES)
```
Comment (Majkl-J) [inline code/modules/surgery/organs/external/tails.dm:124]: The animation on these is still broken but I didn't notice (Spines disappear now), Soothes thankfully brought it up.

Will fix asap

Comment (Majkl-J) [reply code/modules/surgery/organs/external/tails.dm:124]: Mind you this is still better than them not animating

**Comment by SilvernKell** [comment]
Bless you

File: code/modules/surgery/organs/external/tails.dm (hunk 1 above)
Comment (Majkl-J) [reply code/modules/surgery/organs/external/tails.dm:124]: Ok I looked through this. The `build_icon_state()` now works correctly, updating the icon state... but something still makes the spines disappear????

**Comment by nevimer** [comment]
it does not really work right... might be fixed by an upstream merge

**Pull request closed without merging**